test/utils/test-constants.js

```diff
@@ -3,8 +3,8 @@
 var path = require('path');
 
 // Input/output relative paths
-var inputRelative = './fixtures/';
-var outputRelative = './out-fixtures/';
+var inputRelative = './fixtures';
```

Comment:
Do the tests expect/mention ending in a separator?  If so, we need to update descriptions